store/sql_channel_store.go

```diff
@@ -37,6 +38,7 @@ func NewSqlChannelStore(sqlStore *SqlStore) ChannelStore {
 
 func (s SqlChannelStore) UpgradeSchemaIfNeeded() {
 	s.CreateColumnIfNotExists("Channels", "ExtraUpdateAt", "TotalMsgCount", "bigint(20)", "0")
+	s.CreateColumnIfNotExists("Channels", "CreatorId", "ExtraUpdateAt", "varchar(26)", "")
```

Comment:
This won't work in Postgres since CreateColumnIfNotExists just returns early if it's running Postgres. Either we need to make that function work in all databases or we need to come up with a different way to make changes to the schema.